### PR TITLE
CreateIndicatorsFromSTIX script+

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/21333

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [X] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/stixparser-to-context/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.yml)

### CODE REVIEW

File: Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+def main():
+    args = demisto.args()
+    entry_id = args.get("entry_id", "")
+    file_path = demisto.getFilePath(entry_id).get('path')
+    if not file_path:
+        return_error("Could not find file for entry id {}.".format(entry_id))
+    with open(file_path) as file:
+        file_txt = file.read()
+
+    contents = demisto.executeCommand("StixParser", {"iocXml": file_txt})[0].get("Contents")
```
Comment (anara123) [inline Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py]: validate that execute command doesn't returns you an error via `is_error`

File: Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    data = json.loads(contents)
+    indicators = [
+        {
+            "type": stix_struct_to_indicator.get(indicator.get("indicator_type")),
+            "value": indicator.get("value"),
+            "reputation": score_to_reputation(indicator.get("score")),
```
Comment (anara123) [inline Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py]: can we provide also the vendor?

**Comment by anara123** [comment]
@jochman seems like your build is failing
```
CreateIndicatorFromSTIXTest failed with error/s
Playbook CreateIndicatorFromSTIXTest has failed:
Failed: playbook: CreateIndicatorFromSTIXTest failed
```

**Comment by anara123** [comment]
please fix so we can merge to use it in https://github.com/demisto/content/pull/5491

**Comment by jochman** [comment]
> @jochman seems like your build is failing
> 
> ```
> CreateIndicatorFromSTIXTest failed with error/s
> Playbook CreateIndicatorFromSTIXTest has failed:
> Failed: playbook: CreateIndicatorFromSTIXTest failed
> ```

It's a bug in the build.
image
Clearly, the task is not what in the photo (It's getting STIX1 file).

File: Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py (hunk 2 above)
Comment (jochman) [reply Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.py]: added "source"

**Comment by kirbles19** [comment]
@jochman Done

File: TestPlaybooks/playbook-CreateIndicatorFromSTIXTest.yml [hunk 3]
```diff
@@ -186,13 +186,13 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quiet: false
-  "5":
-    id: "5"
-    taskid: "6"
+  "deletecontextonstix":
```
Comment (anara123) [inline TestPlaybooks/playbook-CreateIndicatorFromSTIXTest.yml]: you shouldnt change the ids of the PlaybookTask only the taskid and the inside the task
image

Comment (jochman) [reply TestPlaybooks/playbook-CreateIndicatorFromSTIXTest.yml]: it's working, it's only for the testplaybook.
let's leave it that way and merge.

**Pull request merged**